src/user-guide/interventions/index.rst

```diff
@@ -5,8 +5,8 @@ Interventions
 
 Sometimes things don't go to plan!
 
-So Cylc allows you to take manual control of your workflow whilst it's running
-allowing you to do things like edit a task's configuration, re-run a section
+So Cylc allows you to take manual control of your workflow whilst it's running,
+to do things like edit a task's configuration, re-run a section
```

Comment:
If we can get rid of "do things like" and make the list happily comprehensive I'd be happier.

```suggestion
Cylc allows you to take manual control of your workflow whilst it's running,
to do things like edit a task's configuration or re-run a section
```

Reply:
I was trying to keep this PR strictly to "explanatory info for the examples", rather than a more general revision (note there's a "whilst" in there too 😁 ) ... but I tend to agree with you on that point.